app/queries/servers/post.ts

```diff
@@ -216,3 +218,53 @@ export const observeSavedPostsByIds = (database: Database, postIds: string[]) =>
         switchMap((prefs) => of$(new Set(prefs.map((p) => p.name)))),
     );
 };
+
+export const getIsPostPriorityEnabled = async (database: Database) => {
+    const featureFlag = await getConfigValue(database, 'FeatureFlagPostPriority');
+    const cfg = await getConfigValue(database, 'PostPriority');
+    return featureFlag === 'true' && cfg === 'true';
+};
+
+export const getIsPostAcknowledgementsEnabled = async (database: Database) => {
+    const cfg = await getConfigValue(database, 'PostAcknowledgements');
+    return cfg === 'true';
+};
+
+export const observeIsPostPriorityEnabled = (database: Database) => {
+    const featureFlag = observeConfigBooleanValue(database, 'FeatureFlagPostPriority');
+    const cfg = observeConfigBooleanValue(database, 'PostPriority');
+    return combineLatest([featureFlag, cfg]).pipe(
+        switchMap(([ff, c]) => of$(ff && c)),
+        distinctUntilChanged(),
```

Comment:
this is fine but you can also use `combineLatestWith` the difference is that `combineLatest` will emit every time any of the source observables in the list emits a change while `combineLatestsWith` subscribes to all sources and once all sources emit at least one value, all the values will be emitted as an array. After that, every time any sources emits a value, all of the latest values are emitted as an array.

Why is that important, say featureFlag emits first but `cfg` doesn't, this observable is emitting `false` as a result of checking `true && undefined` when `cfg` then returns `true` this observable then emits again